src/source-updater.js

```diff
@@ -322,7 +322,15 @@ export default class SourceUpdater extends videojs.EventTarget {
   }
 
   buffered() {
-    return buffered(this.videoBuffer, this.audioBuffer);
+    if (this.audioBuffer && !this.videoBuffer) {
```

Comment:
The only unique functionality in that file what using audio or video buffered, if we only had a buffer for one or the other.